addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -134,7 +134,7 @@ export class KanbanRenderer extends Component {
                 }
 
                 // Open first link
-                const firstLink = target.querySelector("a, button");
+                const firstLink = target.querySelector(".oe_kanban_global_click, a, button");
```

Comment:
Sorry, but can you also update the commit message since we will do the fix in `web`? 🙂 

You can explain before your changes and the fix your commit does, that is something like:
```txt
Before this commit, when the user selects a kanban card with the arrows 
key navigation in the project/task kanban view, the focus will be on the 
first `button/a` element found in the kanban card and so it could be 
something else then the classic global click (when the user clicks on 
kanban card).

This commit changes the focus to first select the element containing `oe_kanban_global_click` to click on it to have the same than the 
global click or select the first `a` or `button` in the kanban card.
```

It is just an idea, you can indeed update the suggestion. 🙂